meson_options.txt

```diff
@@ -2,6 +2,7 @@ option('with-liburing', type: 'boolean', value: true)
 option('with-libaio', type: 'boolean', value: true)
 option('with-spdk', type: 'boolean', value: true)
 option('with-fio', type: 'boolean', value: true)
+option('with-ramdisk', type: 'boolean', value: true)
```

Comment:
The ``with-x`` has so far been used for things requiring strong third-party dependencies which are retrieved via meson/wrap/subproject. Since ``memdisk`` is an entirely self-contained feature, then it should go into the other options defined below. For example:
```
option('be_ramdisk', type: 'boolean', value: true)
```

Reply:
Done